src/components/ui/Text.tsx

```diff
@@ -75,10 +80,11 @@ const linkText = (
   extraSmall?: boolean,
   bold?: boolean,
   gradientFrom?: TGradientColorStops,
-  gradientTo?: TGradientColorStops
+  gradientTo?: TGradientColorStops,
+  underline?: boolean
 ) =>
   classnames(
-    textDecoration('no-underline'),
+    textDecoration({ underline: underline }),
```

Comment:
Can we do `{underline}` here? Or it will work like boolean, yeah, probably we can't

Reply:
that's smart! ty, fixed